### PR TITLE
DEV: Improve s3:ensure_cors_rules logging

### PR DESCRIPTION
**Pull request by martin-brennan**

### CODE REVIEW

File: lib/s3_cors_rulesets.rb [hunk 1]
```diff
@@ -26,6 +26,10 @@ class S3CorsRulesets
     max_age_seconds: 3000
   }.freeze
 
+  RULE_STATUS_SKIPPED = "rules_skipped_from_settings".freeze
+  RULE_STATUS_EXISTED = "rules_already_existed".freeze
+  RULE_STATUS_APPLIED = "rules_applied".freeze
```
Comment (CvX) [inline lib/s3_cors_rulesets.rb:31]: `# frozen_string_literal: true` makes these freezes redundant, doesn't it?

Comment (martin-brennan) [reply lib/s3_cors_rulesets.rb:31]: :cold_face: :cold_face: :cold_face: yes it does, I will remove :) Forget about that comment, I just add it mindlessly now.

**Comment by CvX** [comment]
Also looks like specs need to be updated? 😃

**Comment by martin-brennan** [comment]
Yes I see, sorry kind of jumped the gun on review here :sweat_smile: Who knew changing a few log lines would cause trouble?

**Review by CvX — APPROVED** [review]

**Pull request merged**